hadoop-tools/hadoop-azure/src/test/java/org/apache/hadoop/fs/azurebfs/ITestAzureBlobFileSystemDelete.java

```diff
@@ -302,4 +309,109 @@ public void testDeleteIdempotencyTriggerHttp404() throws Exception {
     mockStore.delete(new Path("/NonExistingPath"), false, getTestTracingContext(fs, false));
   }
 
+  @Test
+  public void testDeleteImplicitDir() throws Exception {
```

Comment:
How is this delete of implicit directory if mkdirs is called to create the directory ?

Reply:
doing 
```
fs.getAbfsClient().deleteBlobPath(new Path("/testDir/dir1"),
        null, Mockito.mock(TracingContext.class));
```
to make it implicit.